src/layer/base.ts

```diff
@@ -138,6 +138,9 @@ class Base implements EtroObject {
    */
   seek (time: number): void {
     this._currentTime = time
+    if (!this._currentTime) {
+      this.active = false
```

Comment:
Instead of directly setting `active` to false, call `this.stop()`. Some layers add custom stop logic to this method